src/databricks/labs/ucx/install.py

```diff
@@ -120,6 +121,9 @@ def deploy_schema(sql_backend: SqlBackend, inventory_schema: str):
             functools.partial(table, "udfs", Udf),
             functools.partial(table, "logs", LogRecord),
             functools.partial(table, "recon_results", ReconResult),
+            functools.partial(
+                table, "direct_file_system_access_in_paths", DirectFsAccess
```

Comment:
```suggestion
                table, "directfs_in_workspace", DirectFsAccess
```
please pick a shorter name, as it's abnormally longer than others

Reply:
done, sticking to .`..paths` because `queries` also belong to workspace